### PR TITLE
Add query for a person's upcoming events

### LINKED ISSUE #557: Query for upcoming events for a person

### PR DESCRIPTION
**Pull request by wushenshen**
Closes #557

### CODE REVIEW

File: api/graphql/makeModels.js [hunk 1]
```diff
@@ -111,6 +111,7 @@ export default async function makeModels (userId, isAdmin) {
         'memberships',
         'moderatedCommunityMemberships',
         'locationObject',
+        {events: {querySet: true}},
```
Comment (tibetsprague) [inline api/graphql/makeModels.js:114]: i think we should probably call this eventsAttending

Comment (lorenjohnson) [reply api/graphql/makeModels.js:114]: I agree and would want this reflected in the model too, if that is in fact what is being queried. I'm not actually clear. Is the `User#events` relation only pulling events to which I'm attending or also events I've merely created? 

If I remember correctly I'm always added to the event group ("attending") when I create an event. If that is the case I think the query and this naming is correct.

**Review by lorenjohnson — APPROVED** [review]
👍  LGTM --- pending clarification on the what `User#events` queries. If it's returning all events regardless of whether I created it and/or am attending, then we're good. If there is a distinction here we're missing then I think it needs another look.

File: api/graphql/makeModels.js (hunk 1 above)
Comment (wushenshen) [reply api/graphql/makeModels.js:114]: @lorenjohnson That's correct, the event creator is also added to the event group, and therefore "attending"

**Comment by wushenshen** [comment]
> 👍 LGTM --- pending clarification on the what `User#events` queries. If it's returning all events regardless of whether I created it and/or am attending, then we're good. If there is a distinction here we're missing then I think it needs another look.

No distinction - all events regardless of whether the user created it and/or is attending.

File: api/graphql/makeModels.js (hunk 1 above)
Comment (tibetsprague) [reply api/graphql/makeModels.js:114]: @wushenshen so lets call it eventsAttending then, right? I think if a creator of an event says they are no longer attending the event it should disappear from the list on their profile

**Comment by wushenshen** [comment]
@tibetsprague updated the naming and logic to pull only upcoming events the person is attending. fyi @lorenjohnson

**Pull request merged**